### PR TITLE
PFE Markdown fork | Adds schema and storybook utils to demo

### PR DESCRIPTION
**Pull request by castastrophe**
@kylebuch8 I created a fork that adds a schema, updates the storybook file to use it, and adds support for the context variables which can influence the text color if the markdown appears on a dark background.

### CODE REVIEW

File: .storybook/test-theme.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.PFElement {
```
Comment (starryeyez024) [inline .storybook/test-theme.css]: can this become 
`@import "https://static.redhat.com/libs/redhat/redhat-theme/latest/advanced-theme.css";`

Comment (castastrophe) [reply .storybook/test-theme.css]: I think we discussed this in person already but we want to showcase the theme concept but not link to the Red Hat theme in the open source project.

Comment (castastrophe) [reply .storybook/test-theme.css]: This code is the minimum to make broadcast work without overriding local styles.

File: elements/pfe-markdown/demo/index.html [hunk 2]
```diff
@@ -9,6 +9,9 @@
     </noscript>
 
     <link href="../../pfelement/pfelement.min.css" rel="stylesheet">
+    
+    <!-- Test using the Red Hat theme -->
+    <link href="https://static.redhat.com/libs/redhat/redhat-theme/latest/advanced-theme.css" rel="stylesheet">
```
Comment (castastrophe) [inline elements/pfe-markdown/demo/index.html]: @castastrophe Note to self to remove this.

File: elements/pfe-markdown/src/pfe-markdown.scss [hunk 3]
```diff
@@ -6,3 +6,11 @@
 :host([hidden]) {
   display: none;
 }
+
+:host([on="dark"]) {
```
Comment (kylebuch8) [inline elements/pfe-markdown/src/pfe-markdown.scss]: @castastrophe Are we still using `on` attributes?

**Pull request merged**